src/vs/vscode.d.ts

```diff
@@ -1610,9 +1610,24 @@ declare module 'vscode' {
 		scheme?: string;
 
 		/**
-		 * A glob pattern, like `*.{ts,js}`.
+		 * Either a absolute glob pattern, like `**∕*.{ts,js}` or a [relative pattern](#RelativePattern).
 		 */
-		pattern?: string;
+		pattern?: string | RelativePattern;
+	}
+
+	class RelativePattern {
+
+		/**
+		 * A base to which the pattern will be matched against relatively.
+		 */
+		readonly base: Uri;
```

Comment:
On a second thought this could also just be a string, right? We have the sugar ctor but technically we aren't globbing against a URI but against it path/fspath?

Reply:
@jrieken yes it should be a string because this is about file paths, I changed that via https://github.com/Microsoft/vscode/pull/34695/commits/f370693f0a69b2940b816644471a14ff3eb49ce4